src/app/core/market/api/category/category.service.ts

```diff
@@ -38,4 +38,18 @@ export class CategoryService {
   search(searchString: string): Observable<any> {
     return this.market.call('category', ['search', searchString]);
   }
+
+  getCategoryByCategoryId(categories: any[], categoryId: number)  {
```

Comment:
Hmmm, this seems a bit clunky..
Why is this required? And can we get the categories through list(à-) instead of passing them?

Reply:
I can't even ctrl+f find the usage of this function, is this dead code?